pkg/volume/rbd/rbd.go

```diff
@@ -29,17 +29,20 @@ import (
 )
 
 // This is the primary entrypoint for volume plugins.
-func ProbeVolumePlugins() []volume.VolumePlugin {
-	return []volume.VolumePlugin{&rbdPlugin{nil, exec.New()}}
+func ProbeVolumePlugins(volumeConfig volume.VolumeConfig) []volume.VolumePlugin {
+	return []volume.VolumePlugin{&rbdPlugin{nil, exec.New(), newRecycler, volumeConfig}}
 }
 
 type rbdPlugin struct {
-	host volume.VolumeHost
-	exe  exec.Interface
+	host            volume.VolumeHost
+	exe             exec.Interface
+	newRecyclerFunc func(*volume.Spec, volume.VolumeHost, volume.VolumeConfig) (volume.Recycler, error)
```

Comment:
@markturansky, why is `newRecyclerFunc` here? IMO it always leads to `newRecycler`. I would remove it from all the plugins with recycler (in a separate PR).